pkg/notes/adaptor/presenter/reacton.ts

```diff
@@ -1,7 +1,7 @@
 import { Result } from '@mikuroxina/mini-fn';
 
 export type ReactionError = {
-  code: 400 | 404;
+  code: 400 | 404; // 400: Bad Request / 404: Not Found
```

Comment:
```suggestion
  /** 400: Bad Request / 404: Not Found */
  code: 400 | 404;
```
ドキュメントコメントは (1 行であれば) このように `/** ... */` と囲って記述してください.